projections/weather/spec/weather/endpoint_provider_spec.rb

```diff
@@ -7,5 +7,32 @@
 module Weather
   describe EndpointProvider do
     subject { EndpointProvider.new }
+
+    context "Endpoint set" do
```

Comment:
Surprised that rubocop didn't yell about double quotes here.

Reply:
It does :-(. However, this is arbitrary input so could contain characters that require double quotes.  We could add a view method called `quote` that woudl inspect a given string and decide how to quote it.... but I'm not 100% sure thats worth the effort vs say, running rubocop as a "polish"